### DISCUSSION TITLE
Jetbrains Space Integration

### DISCUSSION THREAD

**Discussion by beisenherz** [jreleaser/jreleaser#1426] [Ideas]
I personally _believe_ that JetBrains Space might be one of the greatest team tools at the market, and it might be the future of future development. 
JReleaser could implement the space SDK and allow the following features:
- Release Notes in Documents/ Blogs
- Releasing to Packages
- Notifications in Channels and/ or PNs

Also we could add a sample for using JReleaser in the Space CI. Would that be of any interest?

If this feature is wanted, I'd be happy to help and assist with the implementation and documentation.

Looking forward to hear your feedback.

**Comment by aalmiray** [comment]
Sounds good to me. I'd suggest starting with a guide for running JReleaser on Space CI. Following up with announce, release, and packaging/upload in that order as that goes from simple to complex.
